loki/ir/tests/test_ir_nodes.py

```diff
@@ -148,6 +154,14 @@ def test_conditional(scope, n, a_i):
     )
     assert not cond.body and cond.else_body == (assign, assign, assign)
 
+    # Test auto-casting with unnamed constructor args
+    cond = ir.Conditional(condition)
+    assert cond.body == () and not cond.else_body
```

Comment:
Pylint doesn't like `cond.body == ()`, maybe 
```suggestion
    assert isinstance(cond.body, tuple) and not (cond.body or cond.else_body)
```

Reply:
Same here, `cond.body is ()` should do to appease pylint...